zabbix_auto_config/processing.py

```diff
@@ -225,7 +225,11 @@ def __init__(self, name, state, db_uri, host_modifier_dir):
     def get_host_modifiers(self):
         sys.path.append(self.host_modifier_dir)
 
-        module_names = [filename[:-3] for filename in os.listdir(self.host_modifier_dir) if filename.endswith(".py")]
+        try:
+            module_names = [filename[:-3] for filename in os.listdir(self.host_modifier_dir) if filename.endswith(".py")]
+        except FileNotFoundError:
+            logging.error("Host modififier directory %s does not exist.", host_modifier_dir)
```

Comment:
Typo? Should be `self.host_modifier_dir`?

Reply:
Good catch, thanks!